src/context/hooks/useAuth.ts

```
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import {
+  AuthenticatedUser,
+  AuthenticationResponse,
+} from '../../commons/interfaces'
+import { api } from '../../libs/axios'
+
+export function useAuth() {
+  const [authenticated, setAuthenticated] = useState(false);
```

ESLint

[error] Delete `;`

(prettier/prettier)